README.md

```diff
@@ -1,29 +1,11 @@
-# Sc2reporter
-Finalized version of the SarsCov2 Reporter app, dubbed sc2reporter.
+# docker-sc2reporter v2
+Dockerized version of the sc2reporter program. But version 2.
 
-
-# Usage
-
-TLDR:
-have docker-compose installed on your system and run following to get a demo with test data going
+## To run with dummy data:
+Make sure you have docker-compose installed on your system, and then
 ```bash
 git clone https://github.com/genomic-medicine-sweden/docker-sc2reporter
 cd docker-sc2reporter
-docker-compose -f docker-compose-demo.yml up
+git checkout add-react-app
```

Comment:
I would not have references to a branch, while the code is not merge you already gotta be in the branch, and once it is merged you won't need it.